src/libraries/System.Collections/src/System/Collections/Generic/CollectionExtensions.cs

```diff
@@ -7,8 +7,7 @@ namespace System.Collections.Generic
 {
     public static class CollectionExtensions
     {
-        [return: MaybeNull]
-        public static TValue GetValueOrDefault<TKey, TValue>(this IReadOnlyDictionary<TKey, TValue> dictionary, TKey key)
+        public static TValue? GetValueOrDefault<TKey, TValue>(this IReadOnlyDictionary<TKey, TValue> dictionary, TKey key)
         {
             return dictionary.GetValueOrDefault(key, default!);
```

Comment:
Can we remove the `!` from `default` here? I guess `ReadOnlyDictionary<TKey, TValue>.GetValueOrDefault` should accept `TValue?` for the default value?

Reply:
If someone wants to pass a nullable TValue, TValue should be specified as nullable :)

Reply:
Ah right, generics 😄